### DISCUSSION TITLE
Reshape tool

### DISCUSSION THREAD

**Discussion by UtterDonkey** [SVG-Edit/svgedit#933] [Q&A]
Is there a tool for editing/creating points in a path similar to the reshape tool in the scratch costume editor (https://scratchfoundation.github.io/scratch-paint)?

**Comment by jfhenon** [comment]
You can do it with SVGEdit if you click on a path already selected, you will turn into edit path mode. 
<img width="482" alt="image" src="https://github.com/SVG-Edit/svgedit/assets/20402845/b5468a22-f912-4975-9800-0a883e7f29ef">

**Reply by UtterDonkey** [reply]
But I'm not able to click a part of the path to insert a new point at.

**Reply by jfhenon** [reply]
You have to click on the toolbar to add points to the path.
<img width="67" alt="image" src="https://github.com/SVG-Edit/svgedit/assets/20402845/5b580fea-c822-4b00-b584-41e4577fea8d">

**Reply by UtterDonkey** [reply]
I mean points on an already existing shape.

**Reply by jfhenon** [reply]
You can only add point on an existing  path. If you want to do it on a rectangle for example, you have to convert the rectangle to  a path using the toolbar:
<img width="133" alt="image" src="https://github.com/SVG-Edit/svgedit/assets/20402845/3a6c8a80-9419-4737-837e-c9774f7cc29b">

**Reply by UtterDonkey** [reply]
But how do you add a point onto an existing path? Go to https://scratchfoundation.github.io/scratch-paint create a shape, select the second tool then click any part of the edge of the shape and you're able to insert a point anywhere on the shape and reshape it.

**Reply by UtterDonkey** [reply]
<img width="67" alt="289291164-5b580fea-c822-4b00-b584-41e4577fea8d" src="https://github.com/SVG-Edit/svgedit/assets/68461677/f1e7fc16-dcd5-4fc0-8e87-eea9fcb1a507">
Just adds a point halfway along, is it possible to add one wherever I click?
